### PR TITLE
Hotfix/essential details for reindex build

### PR DESCRIPTION
**Pull request by redhug1**
### What

Add details of where to get info to be used in building reindex

### How to review

Work through the details to check correctness.

### Who can review

Anyone

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,57 +36,58 @@ environment variables, or with a link to a configuration guide.
...
 
 ## Running Bulk Indexer
 
 The Bulk Indexer creates an Elastic Search 7.10 index with the alias "ons". It loads data from zebedee, and the dataset API, into the ES 7.10 index.
 
-#### Locally
+### Locally
 
-###### Prerequisites
+#### Prerequisites 2
```
Comment (lindenmckenzie) [inline README.md:82]: Maybe 'Local prerequisites' or 'Prerequisites for running locally' might be more descriptive.

File: README.md [hunk 2]
```diff
@@ -150,6 +192,6 @@ See [CONTRIBUTING](CONTRIBUTING.md) for details.
 
 ### License
 
-Copyright © 2016-2022, Office for National Statistics (https://www.ons.gov.uk)
+Copyright © 2016-2022, Office for National Statistics (<https://www.ons.gov.uk>)
```
Comment (lindenmckenzie) [inline README.md:195]: 2023 here I think.

**Review by lindenmckenzie — COMMENTED** [review]
Couple of very minor comments here Rhys - I can see you're doing this as a hotfix, is this because there's features in the develop branch pending? If you want to repoint to develop then I'm happy to take it and release it all today to keep the live apps up to date.

File: README.md (hunk 1 above)
Comment (redhug1) [reply README.md:82]: done

File: README.md (hunk 2 above)
Comment (redhug1) [reply README.md:195]: done

**Review by lindenmckenzie — APPROVED** [review]

**Pull request merged**